tvb_storage/tvb/storage/h5/file/hdf5_storage_manager.py

```diff
@@ -66,13 +66,12 @@ class HDF5StorageManager(object):
     __hfd5_file = None
 
     TVB_ATTRIBUTE_PREFIX = "TVB_"
-    ROOT_NODE_PATH = "/"
     BOOL_VALUE_PREFIX = "bool:"
     DATETIME_VALUE_PREFIX = "datetime:"
     DATE_TIME_FORMAT = '%Y-%m-%d %H:%M:%S.%f'
     LOCKS = {}
 
-    def __init__(self, storage_folder, file_name, buffer_size=600000):
+    def __init__(self, storage_folder, file_name):
```

Comment:
I suggest we keep here the full path as parameter instead of separate folder and file_name

Reply:
In that case do the concatenation inside each of the StorageInterface methods that call methods from this class? Or before the StorageInterface method calls? Also, I guess in that case inside of the constructor we would only be testing the existence of the full path.

Reply:
It seems to me that you can avoid the concatenation. If you look through the usages of the methods in StorageInterface, in most of the places there is a need to do a split of the full path in order to get the folder and file_name, just to send them as separate parameters here. We could avoid many of those path splits, too.